usuarios/delete-usuarios.spec.js

```diff
@@ -0,0 +1,98 @@
+const faker = require('faker')
+const testServer = require('../utils/testServer')
+const { postUser } = require('../utils/user')
+const { getAdminUserById } = require('../utils/user')
+
+const rotaUsuarios = '/usuarios'
+const rotaLogin = '/login'
+const rotaProdutos = '/produtos'
+const rotaCarrinhos = '/carrinhos'
+
+// beforeAll(async () => {
+//   const { _id } = await postUser()
+
+//   const { email, password } = await getAdminUserById(_id)
+
+//   const { body } = await testServer.post(rotaLogin).send({ email, password })
+
+//   auth.token = body.authorization
+// })
+
+describe('Deletar um usuário através da rota DELETE', () => {
+  it('Deletar um usuário com id não encontrado', async () => {
+    const response = await testServer.delete(`${rotaUsuarios + '/123'}`).send({
+      nome: faker.name.firstName() + ' ' + faker.name.lastName(),
+      email: faker.internet.email(),
+      password: faker.internet.password(),
+      administrador: `${faker.random.boolean()}`
+    })
```

Comment:
a rota de delete não precisa passar o body
```suggestion
    const response = await testServer.delete(`${rotaUsuarios + '/123'}`)
```